cfme/tests/automate/custom_button/__init__.py

```diff
@@ -91,3 +91,12 @@ class TextInputDialogView(View):
     def is_displayed(self):
         # This is only for wait for view
         return self.submit.is_displayed and self.service_name.is_displayed
+
+
+class CustomButtonSSUIDropdwon(Dropdown):
+    """This is workaround for custom button Dropdown in SSUI item_enabled method"""
+
+    def item_enabled(self, item):
```

Comment:
Optional : 
This can be renamed to is_item_enabled or is_enabled

Reply:
I am overwriting method of Dropdown widget.